components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -913,6 +913,11 @@ interface TxService {
   Reset();
 };
 
+union MessageToSignUnion {
+  string? message_str;
+  array<uint8>? message_bytes;
```

Comment:
No need for ? here. The message should either be null entirely, or one of them should have value here.